scripts/ci/run-ci-tests.sh

```diff
@@ -291,5 +285,8 @@ make -C test/others/libcriu run
 # external namespace testing
 make -C test/others/ns_ext run
 
+# Skip all further tests when running with GCOV=1
+[ -n "$GCOV" ] && exit 0
```

Comment:
The comment has to explain why we need to skip further tests. This sounds strange because we want to know the coverage of all tests.

Reply:
compel does not know about GCOV=1 and does not set the corresponding compiler flags. It just completely failed with:
```
2021-02-01T08:41:22.0702344Z gcc -O2 -g -Wall -Werror -o victim victim.c
2021-02-01T08:41:22.1380429Z gcc -O2 -g -Wall -Werror -c -Wstrict-prototypes -fno-stack-protector -nostdlib -fomit-frame-pointer -ffreestanding -fpie -I ../../../compel/include/uapi -o parasite.o parasite.c
2021-02-01T08:41:22.1853761Z ld -r -z noexecstack -T ../../../compel/arch/x86/scripts/compel-pack.lds.S -o parasite.po parasite.o ../../../compel/plugins/std.lib.a ../../../compel/plugins/fds.lib.a
2021-02-01T08:41:22.1880577Z ../../../compel/compel-host hgen -o parasite.h -f parasite.po
2021-02-01T08:41:22.1943862Z gcc -O2 -g -Wall -Werror -I ../../../compel/include/uapi -o spy spy.c ../../../compel/libcompel.a
2021-02-01T08:41:22.3370185Z /usr/bin/ld: ../../../compel/libcompel.a(log.o): in function `_GLOBAL__sub_I_00100_0_log.c':
2021-02-01T08:41:22.3371919Z log.c:(.text.startup+0x8): undefined reference to `__gcov_init'
2021-02-01T08:41:22.3373190Z /usr/bin/ld: ../../../compel/libcompel.a(log.o): in function `_GLOBAL__s
```
and much more.

I updated the comment to make it clear that compel fails with it and that it is a place for tests which do not work with GCOV=1.